config/locales/account/confirmations.en.yml

```diff
@@ -6,26 +6,47 @@ en:
-      </ul>
-    heading: Confirm your email address
-    instruction_one: We’ve sent an email to
-    instruction_two: 'You need to click on the confirmation link in the email to:'
-    new_user_instruction_list: |
-      <ul class="govuk-list govuk-list--bullet">
-        <li>finish creating your account</li>
-        <li>start receiving email updates about Brexit</li>
-      </ul>
-    re_register_instructions: |
-      <p class="govuk-body">We can <a class="govuk-link" href="%{resend_confirmation_link}" data-module="gem-track-click" data-track-category="account-create" data-track-action="confirm-email" data-track-label="resend-email">send the confirmation email again</a> if you did not get it.</a>
-      <p class="govuk-body">You can <a class="govuk-link" href="%{retry_link}" data-module="gem-track-click" data-track-category="account-create" data-track-action="confirm-email" data-track-label="different-email">use a different email address</a> if your email address is not correct.</p>
-    re_register_subtitle: If you did not get the email or want to use a different address
+    heading:
+      confirm_change: Finish updating your email address
+      finish_creating: Finish creating your account
+    help:
+      heading: If you did not get the confirmation email
+      instructions:
+        confirm_change: <p class="govuk-body">We can <a class="govuk-link" href="%{resend_confirmation_link}" data-module="gem-track-click" data-track-category="account-create" data-track-action="confirm-email" data-track-label="resend-email">send the confirmation email again</a> if you did not get it.</a></p>
+        finish_creating: |
+          <p class="govuk-body">We can <a class="govuk-link" href="%{resend_confirmation_link}" data-module="gem-track-click" data-track-category="account-create" data-track-action="confirm-email" data-track-lab>
+          <p class="govuk-body">You can <a class="govuk-link" href="%{retry_link}" data-module="gem-track-click" data-track-category="account-create" data-track-action="confirm-email" data-track-label="different>
+      title: Help if you did not get the confirmation email
+    instructions:
+      confirm_change:
+        details:
+          brexit: |
+            <p class="govuk-body">You need to confirm your new email address.</p>
+            <p class="govuk-body">We’ve sent an email to <strong>%{email}</strong>. Click on the confirmation link in the email to:</p>
+            <ul class="govuk-list govuk-list--bullet">
+              <li>finish updating the email address for your GOV.UK account</li>
+              <li>get email updates about Brexit sent to your new email address</li>
+            </ul>
+          dashboard: |
+            <p class="govuk-body">You need to confirm your new email address.</p>
+            <p class="govuk-body">We’ve sent an email to <strong>%{email}</strong>. Click on the confirmation link in the email to finish updating the email address for your GOV.UK account.</p>
```

Comment:
This can be simplified due to https://github.com/alphagov/govuk-account-manager-prototype/pull/911/files#r676458169